packages/graphql/src/translate/queryAST/factory/FilterFactory.ts

```diff
@@ -522,12 +509,13 @@ export class FilterFactory {
         return this.createPropertyFilter({
             attribute,
             comparisonValue: value,
-            isNot,
+
             operator,
             relationship,
         });
     }
-
+    // TODO: we keep call this Generic filters but maybe we should rename it to something more meaningful
+    // Proposal: TypeSpecificFilters
```

Comment:
:thinking: 

`parseGenericTypeFilters`?

I'm not sure what `TypeSpecific` means

Reply:
This was from a private convo where I asked the team if they have a better name for these new style filters, I was not personally happy with `generic`, this was one proposal received. I believe the reason behind this is that these filters are generated depending on the field type.
I'll remove the comment anyway